docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+		"sample_barcode": null,
+		"submitter_read_group_id": "dash-1-rg-1"
+	}],
+	"experiment": {
+		"submitter_sequencing_experiment_id": "DASH-SE-1",
+		"experimental_strategy": "WGS",
+		"sequencing_center": "",
+		"platform": "ILLUMINA",
+		"platform_model": null,
+		"sequencing_date": null
+	}
+}
+```
+
+</TabItem>
+<TabItem value="tumour">
+
+```json
+{
+	"studyId": "DASH-CA",
+	"analysisType": {
+		"name": "sequencing_experiment"
+	},
+	"samples": [{
+		"submitterSampleId": "dashsa-1.1",
+		"matchedNormalSubmitterSampleId": "dashsa-1",
+		"sampleType": "Amplified DNA",
+		"specimen": {
+			"submitterSpecimenId": "dashsp-1.1",
+			"specimenType": "Primary tumour",
+			"tumourNormalDesignation": "Tumour",
+			"specimenTissueSource": "Cerebellum"
+		},
+		"donor": {
+			"submitterDonorId": "DASH-1",
+			"gender": "Male"
+		}
+	}],
+	"files": [{
+		"dataType": "Submitted Reads",
+		"fileName": "dash-1-tumour.bam",
+		"fileSize": 24,
+		"fileType": "BAM",
+		"fileMd5sum": "d5fd5482b53ee89e71eda8d69cba4bd9",
+		"fileAccess": "controlled"
+	}],
+	"read_group_count": 1,
+	"read_groups": [{
+		"file_r1": "dash-1-tumour.bam",
+		"file_r2": null,
+		"insert_size": null,
+		"is_paired_end": false,
+		"library_name": "Library-Testing",
+		"platform_unit": "Library-Testing",
+		"read_length_r1": null,
+		"read_length_r2": null,
+		"sample_barcode": null,
+		"submitter_read_group_id": "dash-1-rg-2"
+	}],
+	"experiment": {
+		"submitter_sequencing_experiment_id": "DASH-SE-1.2",
+		"experimental_strategy": "WGS",
+		"sequencing_center": "",
+		"platform": "ILLUMINA",
+		"platform_model": null,
+		"sequencing_date": null
+	}
+}
+```
+
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+See github for the same example of a correctly formatted payload from a [normal sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_normal.json) and a [tumour sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_tumour.json).
+
+Depending on the type of sequencing that was done, the `read_groups` section of the payload will need to be adjusted. The following are examples of correctly formatted `read_groups` that meet [the rules](/docs/submission/molecular-data-prep#file-and-data-validation-rules) for paired-end vs single-end sequencing:
+
+<!---  Tabs start here -->
+
+<Tabs
+  defaultValue="BAM-paired"
+  values={[
+    { label: 'BAM (paired)', value: 'BAM-paired', },
+    { label: 'BAM (single)', value: 'BAM-single', },
+    { label: 'FASTQ (paired)', value: 'FASTQ-paired', },
+    { label: 'FASTQ (single)', value: 'FASTQ-single', },
+  ]
+}>
+<TabItem value="BAM-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+  	"read_group_id_in_bam": "C0HVY.2",
+  	"submitter_read_group_id": "C0HVY.2",
+  	"platform_unit": "74_8a",
+  	"is_paired_end": true,
+  	"file_r1": "test_rg3.bam",
+  	"file_r2": "test_rg4.bam",
+  	"read_length_r1": 150,
+  	"read_length_r2": 16,
```

Comment:
about "read_length_r2": 16,
The number of `16` is pretty odd to me. Can we replace `16` to `150`? 
BWA-MEM wont work well with read length of 16.

Reply:
can it be different than r1?

Reply:
r1 and r2 need not necessarily be the same length.

Reply:
ok i put 125 just to show them `not` the same!